### PR TITLE
add holidays to core standing time

### PR DESCRIPTION
**Pull request by stefansc1**

### CODE REVIEW

**Review by jakobgemassmer — CHANGES_REQUESTED** [review]
Code in util.py works. Holidays are also written into scenario.json, but not treated equally to full_days in the flex_band. The flex_band ignores the "holidays". Holidays are also not consired, when generating synthetic driving profiles.

- [x] Treat holidays equally to full_days in `generate_schedule.py`.
- [ ] Add holidays to `generate.py` (via generate.cfg as in generate_schedule.cfg), so they are consired when creating syntetic driving profiles (like `no_drive_days`).

**Comment by stefansc1** [comment]
- added holidays to generate and generate.cfg (config-option only)
- using datetime.weekday instead of isoweekday (off-by-one). Sunday is now 6 instead of 7
- renamed full_days to no_drive_days
- updated tests
- updated RTD

File: generate_schedule.py [hunk 1]
```diff
@@ -21,7 +21,7 @@ def generate_flex_band(scenario, gcID, core_standing_time=None):
     :param gcID: grid connector ID for which to create this flex band
     :type gcID: string
     :param core_standing_time: core standing time during which flexibility is guaranteed e.g.
-        {"times":[{"start": [22,0], "end":[5,0]}], "full_days":[7]}
+        {"times":[{"start": [22,0], "end":[5,0]}], "full_days":[6]}
```
Comment (jakobgemassmer) [inline generate_schedule.py:24]:
```suggestion
        {"times":[{"start": [22,0], "end":[5,0]}], "no_drive_days":[6]}
```

**Review by jakobgemassmer — CHANGES_REQUESTED** [review]
Just a minor name change in the docstrings of generate_schedule.py. Everything else seems solid.

**Pull request merged**